website/notifications/tasks.py

```diff
@@ -46,9 +48,76 @@ def send_users_email(send_type):
                 )
             remove_notifications(email_notification_ids=notification_ids)
 
+    # Send emails to reviews moderators
```

Comment:
Would breaking these into two separate functions make more sense? Using the name of the function to help delineate what each does?

Reply:
That was my original implementation. But on second thought, I am a bit reluctant to schedule additional periodic tasks using crontab if I don't have to.

Reply:
I don't mean completely separate cronjobs, just two functions run by the same job to clarify better what each is doing